src/Workspaces/Core/Portable/EmbeddedLanguages/LanguageServices/FallbackEmbeddedClassifier.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright(c) Microsoft.All Rights Reserved.Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+using System.Threading;
+using Microsoft.CodeAnalysis.Classification;
+using Microsoft.CodeAnalysis.PooledObjects;
+namespace Microsoft.CodeAnalysis.EmbeddedLanguages.LanguageServices
+{
+    internal class FallbackEmbeddedClassifier : IEmbeddedClassifier
+    {
+        private readonly FallbackEmbeddedLanguage _language;
+
+        public FallbackEmbeddedClassifier(FallbackEmbeddedLanguage language)
+        {
+            _language = language;
+        }
+
+        public void AddClassifications(
```

Comment:
>AddClassifications [](start = 20, length = 18)

It seems this PR doesn't add tests to verify classification. Do we have some infrastructure for that?

Reply:
You are correct.  I will add a bunch to validate.

Reply:
Tests added